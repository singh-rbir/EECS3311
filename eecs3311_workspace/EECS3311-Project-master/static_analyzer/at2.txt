add_division 

reset

add_division
add_integer_constant (5)
add_integer_constant (0)
type_check
evaluate

reset