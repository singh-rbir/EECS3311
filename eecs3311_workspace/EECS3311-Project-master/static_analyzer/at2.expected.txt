  Expression currently specified: ?
  Report: Expression is initialized.
->add_division
  Expression currently specified: (? / nil)
  Report: OK.
->reset
  Expression currently specified: ?
  Report: OK.
->add_division
  Expression currently specified: (? / nil)
  Report: OK.
->add_integer_constant(5)
  Expression currently specified: (5 / ?)
  Report: OK.
->add_integer_constant(0)
  Expression currently specified: (5 / 0)
  Report: OK.
->type_check
  Expression currently specified: (5 / 0)
  Report: (5 / 0) is type-correct.
->evaluate
  Expression currently specified: (5 / 0)
  Report: Error (Divisor is zero).
->reset
  Expression currently specified: ?
  Report: OK.
